__tests__/Unit/utils/urlTransform.test.ts

```diff
@@ -0,0 +1,39 @@
+import {
+    extractRepoName,
+    transformGitHubApiUrl,
+} from '../../../src/utils/urlTransform';
+
+describe('Unit | Util | Transform GitHub Api Url', () => {
+    test('returns proper value with proper input', () => {
+        expect(
+            transformGitHubApiUrl(
```

Comment:
nit: more readable would be to assign the value of the function call to a variable and then asserting.

This separates action and the assertion.